TeamCode/src/main/java/org/firstinspires/ftc/teamcode/PixyCam.java

```diff
@@ -0,0 +1,170 @@
+package org.firstinspires.ftc.teamcode;
+
+import android.util.Log;
+
+import com.qualcomm.robotcore.hardware.I2cDeviceSynch;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+/**
```

Comment:
Where do all these constant values come from? I would put a link in the top comment to the relevant specs pages (I had no idea where they came until reading at least http://cmucam.org/projects/cmucam5/wiki/Porting_Guide#The-serial-protocol and http://docs.lpcware.com/lpc800um/RegisterMaps/i2c/r-Register-description.html, for example)